packages/lib/authOptions.ts

```diff
@@ -205,119 +211,104 @@ export const authOptions: NextAuthOptions = {
           return "/auth/login?error=A%20user%20with%20this%20email%20exists%20already.";
         }
 
-        await prisma.user.create({
-          data: {
+        const prodActionClasses = [
+          {
+            name: "New Session",
+            description: "Gets fired when a new session is created",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "Exit Intent (Desktop)",
+            description: "A user on Desktop leaves the website with the cursor.",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "50% Scroll",
+            description: "A user scrolled 50% of the current page",
+            type: "automatic" as "automatic",
+          },
+        ];
+
+        const prodAttributeClasses = [
+          {
+            name: "userId",
+            description: "The internal ID of the person",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "email",
+            description: "The email of the person",
+            type: "automatic" as "automatic",
+          },
+        ];
+
+        const devActionClasses = [
+          {
+            name: "New Session",
+            description: "Gets fired when a new session is created",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "Exit Intent (Desktop)",
+            description: "A user on Desktop leaves the website with the cursor.",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "50% Scroll",
+            description: "A user scrolled 50% of the current page",
+            type: "automatic" as "automatic",
+          },
+        ];
+
+        const devAttributeClasses = [
+          {
+            name: "userId",
+            description: "The internal ID of the person",
+            type: "automatic" as "automatic",
+          },
+          {
+            name: "email",
+            description: "The email of the person",
+            type: "automatic" as "automatic",
+          },
+        ];
+
+        const registerUserAndSetupTeam = async () => {
+          const userProfile = await createProfileWithProviders({
```

Comment:
I think for most of the fields we can use and extend the input type of `createProfile` and we don't need a separate service for this.
For the account we should just create a new service `createAccount` that we can use to create a new account with the id of the newly created as userId